LICENSE

```diff
@@ -1,4 +1,4 @@
-   Copyright © 2020 Progress Software Corporation and/or one
+   Copyright © 2021 Progress Software Corporation and/or one
```

Comment:
It's already April in 2022 ;)

Reply:
The changes were made in 2021. Nothing new was modified because nobody would approve my original PR. I just need this out the door.

Reply:
Updated since we now have changes for 2022